net-dns/dnscap/dnscap-1.10.3.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Network capture utility designed specifically for DNS traffic"
+HOMEPAGE="https://dnscap.dns-oarc.net/"
+SRC_URI="https://www.dns-oarc.net/files/dnscap/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="ISC"
+IUSE=""
+
+RDEPEND="dev-libs/openssl:=
```

Comment:
Could you add a slot here, I think `:0=` is the correct syntax.

Reply:
changed to use the 0 slot of openssl